crates/mun_codegen/src/ir/type_table.rs

```diff
@@ -103,12 +107,12 @@ impl<'db, 'ink, 't> TypeTableBuilder<'db, 'ink, 't> {
     pub(crate) fn new<'f>(
         db: &'db dyn HirDatabase,
         value_context: &'t IrValueContext<'ink, '_, '_>,
-        intrinsics: impl Iterator<Item = &'f FunctionPrototype>,
+        _intrinsics: impl Iterator<Item = &'f FunctionPrototype>,
```

Comment:
Why do we no longer need these types?

Reply:
I think there is still a bug here, really the only types that need to be stored in the type looking tables are the types that we pass to the new function. All the other types can be referred to by typeids.

I’ll update the builder to only insert the types turkey needed.

Reply:
True, given that they are all primitive types.